tests/qinfo/test_fisher_deprecation.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for the deprecation of the classical and quantum fisher information matrix in the pennylane.qinfo
+"""
```

Comment:
I don't think it's worth adding a new file for this test to be honest, but it's harmless, so optional, non-blocking.